packages/bridge/src/client.ts

```diff
@@ -54,14 +54,14 @@ export class AxelarBridgeClient {
     };
 
     if (isNativeTokenDeposit) {
-      return await contract.deposit(
+      return await contract.deposit?.(
         senderAddress, // tokenSender
         receiverAddress, // carbonReceiver bech32Address
         txParams
       );
     }
 
-    return await contract.deposit(
+    return await contract.deposit?.(
```

Comment:
if we declare the ABI and we're sure deposit exists, we should force unwrap `!` instead of optional chain, since it creates a unnecessary null check requirement for the caller.